pkg/oauth/apis/oauth/validation/validation_test.go

```diff
@@ -467,6 +498,22 @@ func TestValidateAccessTokensUpdate(t *testing.T) {
 			T: field.ErrorTypeInvalid,
 			F: "[]",
 		},
+		"change InactivityTimeoutSeconds to smaller value": {
```

Comment:
done and fixed code to refer to inactivityTimeoutSeconds instead of timeoutsIn ... it be nice if those strings could be derived from code variable names via some macro system rather than have to type them in and get things wrong ...